File: problem/Kotlin/number-of-good-pairs.kt
class Solution {
    fun numIdenticalPairs(nums: IntArray): Int {
        var ans = 0
        for(i in 0..nums.size-1){
            for(j in i+1..num.size-1){
                if(nums[i].equals(nums[j]))
                    ans = ans + 1
            }
        }
        return ans
    }
}

File: problem/C++/number-of-good-pairs.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int l = nums.size();
        int i, j, ans = 0;
        for(i = 0 ; i < l ; ++i)
            for(j = i + 1 ; j < l ; ++j)
                if(nums[i] == nums[j])
                    ++ans;
        return ans;
    }
};  